src/checks/quote_character.rs

```diff
@@ -0,0 +1,124 @@
+use crate::checks::Check;
+use crate::common::*;
+
+pub(crate) struct QuoteCharacterChecker {
+    name: String,
```

Comment:
```suggestion
    name: &'a str,
```
Please, use `&'a str` instead of `String`.